pandas/tests/io/test_excel.py

```diff
@@ -1212,6 +1212,22 @@ class and any subclasses, on account of the `autouse=True`
 class TestExcelWriter(_WriterBase):
     # Base class for test cases to run with different Excel writers.
 
+    def test_excel_sheet_size(self): 
+        breaking_row_count = 2**20 + 1
```

Comment:
can you add the issue number as a comment here

Reply:
done